### PR TITLE
Rename patch set

### PR DESCRIPTION
**Pull request by mkholjuraev**
Renames all occurances of 'Patch set' on the UI to 'Patch template'. There are some places only 'Template' word is used. As the mockups are not updated in detail, I tried to cover all places and think about correct words, where to rename... 

Only wording on the UI should change without effecting functionality.

Please refer to the [mockups](https://marvelapp.com/prototype/hddg13b/screen/86218222) for more information.

### CODE REVIEW

File: src/SmartComponents/PatchSet/PatchSetAssets.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const patchSetRowActions = (openPatchSetEditModal, handlePatchSetDelete)
         }
```
Comment (gabipodolnikova) [inline src/SmartComponents/PatchSet/PatchSetAssets.js:37]: And 'Edit patch set' is staying in row actions

**Review by gabipodolnikova — COMMENTED** [review]
Found one possible missing rename and also the navigation item Patch set should be renamed in the config repo right? :)

File: src/SmartComponents/PatchSet/PatchSetAssets.js (hunk 1 above)
Comment (mkholjuraev) [reply src/SmartComponents/PatchSet/PatchSetAssets.js:37]: @gabipodolnikova thank you for the quick look. Nope, I will rename this line as well :)

Comment (mkholjuraev) [reply src/SmartComponents/PatchSet/PatchSetAssets.js:37]: @gabipodolnikova I have fixed ^. You can find it easly on a new commit.

**Review by gabipodolnikova — APPROVED** [review]
LGTM! Thanks! :)

**Pull request merged**